### PR TITLE
Add SQL logic directly to the schema reference from the docs.

### PR DESCRIPTION
**Pull request by jenningsanderson**
# Description

*Brief description of the business purpose and effect of the pull request.*

- Adds the SQL logic for each theme directly into the schema reference.
- Move descriptions of names, height, and 3d buildings from `osm` folder to the documentation directly. 

# Reference

1. Base/Building SQL logic moved to the data-platform-tf for better testing / integration

# Testing

https://dfhx9f55j8eg5.cloudfront.net/pr/268/schema/concepts/by-theme/base
https://dfhx9f55j8eg5.cloudfront.net/pr/268/schema/concepts/by-theme/buildings
https://dfhx9f55j8eg5.cloudfront.net/pr/268/schema/concepts/names

# Checklist

*Checklist of tasks commonly-associated with schema pull requests. Please review the relevant checklists and ensure you do all the tasks that are required for the change you made.*

1. [N/A] Add relevant examples.
2. [N/A] Add relevant counterexamples.
3. [N/A] Update any counterexamples that became obsolete. For example, if a counterexample uses property `A` but is not intended to test property `A`'s validity, and you made a schema change that invalidates property `A` in that counterexample, fix the counterexample to align it with your schema change.  
4. [X] Update in-schema documentation using plain English written in complete sentences, if an update is required.
5. [X] Update Docusaurus documentation, if an update is required.
6. [x] Review change with Overture technical writer to ensure any advanced documentation needs will be taken care of, unless the change is trivial and would not affect the documentation.

# Documentation Website

*Update the hyperlink below to put the pull request number in.*

[Docs preview for this PR.](https://dfhx9f55j8eg5.cloudfront.net/pr/268/schema/)

### CODE REVIEW

**Review by vcschapp — DISMISSED** [review]

File: docs/schema/concepts/by-theme/base/index.mdx [hunk 1]
```diff
@@ -1,21 +1,70 @@
...
 The base theme has five feature types.
-- `infrastructure` is a feature type that... It has a geometry
-- `land` is a feature type that... It has a Polygon or MultiPolygon geometry.
-- `land_cover` is a feature type that... It has a Polygon or MultiPolygon geometry.
-- `land_use` is a feature type that... It has a Polygon or MultiPolygon geometry.
-- `water` is a feature type that... It has a LineString, MultiLineString, Polygon or MultiPolygon geometry.
+
+- `infrastructure` - Human-made structures (bridges, towers, airports, etc.) from OpenStreetMap
+- `land` - Natural land features from OpenStreetMap
+- `land_cover` Derived from ESA imagery
```
Comment (vcschapp) [inline docs/schema/concepts/by-theme/base/index.mdx:24]: Should there be a hyphen here?

**Review by DavidKarlas — APPROVED** [review]

**Review by danabauer — DISMISSED** [review]
this is a good step toward documenting the daylight to overture transition!
https://github.com/OvertureMaps/tf-developer-advocacy/issues/165

**Review by vcschapp — DISMISSED** [review]
```
         ,..........   ..........,
     ,..,'          '.'          ',..,
    ,' ,'            :            ', ',
   ,' ,'             :             ', ',
  ,' ,'              :              ', ',
 ,' ,'............., : ,.............', ',
,'  '............   '.'   ............'  ',
 '''''''''''''''''';''';''''''''''''''''''
                    '''
```

**Review by atiannicelli — DISMISSED** [review]
Meant to review this earlier. Looks great, but I left one comment.

**Review by vcschapp — APPROVED** [review]
Re-up!

**Review by jwass — APPROVED** [review]

**Pull request merged**